docs/userGuide/reusingContents.md

```diff
@@ -145,6 +155,17 @@ You must use the `safe` filter when using such variables:
 <code>{<span></span>{ right_hand_2 }}</code> {{ icon_arrow_right }} {{glyphicon_hand_right}}
 </div>
 
+**Variables can also be used in element attributes.**
+
+<div class="indented">
+
+{{ icon_example }} Specifying a variable as an attribute:
+
+<code>\<span id="color">red</span>\</code><br>
```

Comment:
The wrong HTML tag is escaped, it didn't render correctly.

Reply:
Also it is not clear that the two different code exist in two different places, especially when you look at this portion in the live preview. Maybe indicate one is in `variables.md` and one is in `index.md`?